custom/icds/tests/test_aww_no_submissions.py

```diff
@@ -0,0 +1,67 @@
+from datetime import date, datetime, timedelta
+from django.test import TestCase
+from mock import patch
+import pytz
+
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.apps.locations.tests.util import make_loc, setup_location_types
+from corehq.apps.users.models import CommCareUser
+from custom.icds.messaging.indicators import AWWSubmissionPerformanceIndicator
+
+
+@patch('custom.icds.messaging.indicators.get_last_submission_time_for_users')
```

Comment:
I think whenever you patch a method you have to patch where it's imported. It always trips me up, but patching esaccessors will work